plugins/module_utils/network/junos/argspec/bgp_global/bgp_global.py

```diff
@@ -64,8 +64,8 @@ def __init__(self, **kwargs):
                     "choices": ["aes-128-cmac-96", "hmac-sha-1-96", "md5"],
                     "type": "str",
                 },
-                "authentication_key": {"type": "str"},
-                "authentication_key_chain": {"type": "str"},
+                "authentication_key": {"type": "str", "no_log": True},
+                "authentication_key_chain": {"type": "str", "no_log": True},
```

Comment:
The `authentication_key_chain`, based on the description in doc seems to be the name of the key chain. This might not be a sensitive information. If that is the case, we can set `no_log: False` for this key.